### DISCUSSION TITLE
DS18x20 on 9.2.0

### DISCUSSION THREAD

**Discussion by swampdogmash** [arendst/Tasmota#10613] [Q&A] [answered]
I've just upgraded from 8.3.1 to 9.2.0 on all my devices (including on one device a custom build). for some reason, on one of the Wemos devices, I now can't get temp data from a DS18x20. I've had a flick through all the release notes - have i missed something about DS18x20 not being supported on a specific pin or in specific configurations? It's currently configured on a generic module that has a number of relays attached, plus switches and then the DS18x20.

![image](https://user-images.githubusercontent.com/35781133/104924911-02b03180-5996-11eb-93ce-564e8c3ca9e0.png)

Not sure what may have caused this but I did see something about DS18x20 on the closed issues but that seemed to refer to a custom build which this isn't (it's using the release binary from the standard OTA repository.

I did try the the latest 9.2.0.3 build but when doing an OTA i couldn't manage to get past being on the minimal firmware.

The device is remotely located and not easily accessible (nor moveable) for the hardware to have moved or become disconnected and it was working before the upgrade - my point is i am confident that (at this stage) that it is not a hardware issue.

**Comment by swampdogmash** [comment, marked as answer]
Just retried the upgrade to 9.2.0.3 and it succeeded this time (suspect an error in the URL i had copied and pasted). Looks like I still have no DS18x20 sensor.

**Reply by barbudor** [reply]
Are you sure to use the full Tasmota.bin(.gz) and not the tasmota-Lite ?

**Reply by swampdogmash** [reply]
thanks for the reply. 

100% I am using the full bin.

![image](https://user-images.githubusercontent.com/35781133/104933980-7c99e800-59a1-11eb-971d-9e28ab2d54d0.png)

URL for upgrade was http://ota.tasmota.com/tasmota/releases/tasmota.bin.gz

![image](https://user-images.githubusercontent.com/35781133/104934028-8facb800-59a1-11eb-9d7e-427d09c0e056.png)

**Reply by barbudor** [reply]
Could you join the Discord server ? It would be more easier to investigate with you in a more interactive fashion
thanks

**Reply by swampdogmash** [reply]
Have you got the exact details?  I have been having trouble finding the correct Discord link for Tasmota, and when I did manage to get somewhere close, none of the dropdowns to enter my birth date would let me select anything. If you could provide the correct link i will most definitely join :)

**Reply by heimchemiker** [reply]
Have you tried the Sensor Build or built your own with the DS18x20 enabled?
